<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryFormDialog</class>
 <widget class="QDialog" name="CategoryFormDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Formulario de categoría</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;
background-color: rgb(218, 213, 207);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Segoe UI Variable</family>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>Definir nueva categoría de productos.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="producCategory_label">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Nombre de categoría: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="categoryName_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
  border: 2px solid  rgb(237, 233, 230);
  border-radius: 15px;

  color: black;
  background-color: rgb(237, 233, 230);
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 10px;
  padding-bottom: 10px;
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acceptCategory_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 20px;
	padding-right: 20px px;
	background-color: rgb(0, 153, 73);
	color: rgb(255, 255, 255);
}

QPushButton:hover {
  background-color: rgb(0, 181, 84);     /* Fondo al pasar el mouse */
  color: #f0f0f0;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(0, 125, 58);     /* Fondo al presionar */
  color: #d0d0d0;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelCategory_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 20px;
	padding-right: 20px px;
	background-color: rgb(232, 54, 18);
	color: rgb(255, 255, 255);
}

QPushButton:hover {
  background-color: rgb(253, 59, 20);     /* Fondo al pasar el mouse */
  color: #f0f0f0;                /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: rgb(208, 48, 16);    /* Fondo al presionar */
  color: #d0d0d0;                /* Color del texto al presionar */
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
